engine/apps/user_management/models/organization.py

```diff
@@ -344,6 +345,22 @@ def get_notifiable_direct_paging_integrations(self) -> "RelatedManager['AlertRec
             .distinct()
         )
 
+    def should_be_considered_for_rbac_permissioning(self) -> bool:
+        """
+        this is sort of a hacky workaround to address a cloud issue we introduced with the accessControlOncall
+        feature flag. The flag is technically enabled for all stacks, but the way in which OnCall used to be
+        reading it (via GCOM config.feature_flags for the stack) made it such that RBAC wasn't actually being
+        enabled for most stacks from the oncall backend perspective. Once we change things to start HEADing
+        the permissions search endpoint, this will effectively turn on RBAC for all orgs.. soo instead lets
+        slowly turn it on via the logic here
+        """
+        cloud_rbac_rollout_percentage = settings.CLOUD_RBAC_ROLLOUT_PERCENTAGE
+
+        # if rbac permissions are already enabled for the org, they're "grandfathered" in
+        if not cloud_rbac_rollout_percentage or self.is_rbac_permissions_enabled:
+            return True
+        return self.id <= math.floor(Organization.objects.count() * cloud_rbac_rollout_percentage)
+
```

Comment:
cc: @gamab

Reply:
this will eventually be removed once we're comfortable rolling this out to all stacks in cloud (ie. we'll slowly ramp up the value of `CLOUD_RBAC_ROLLOUT_PERCENTAGE`)